samples/client/petstore/C/api/UserAPI.c

```diff
@@ -223,14 +259,17 @@ char_t *UserAPI_loginUser(apiClient_t *apiClient, char *Username,
 	                 "GET");
 
 	free(apiClient->dataReceived);
-	free(Username);
-	free(Password);
-	localVarchar = _parseFromJSON(apiClient->dataReceived);
+	free(localVarPath);
+
+	free(username);
+	free(password);
+
+	char_t *localVarchar = User_parseFromJSON(apiClient->dataReceived);
```

Comment:
Unable to differentiate here between string and object. eg. char and pet. isString doesnt work.